ts/test/local/conductor.ts

```diff
@@ -165,10 +167,10 @@ test("Local Conductor - install app with deferred memproofs", async (t) => {
   const appWs = await conductor.connectAppWs(issued.token, port);
 
   let appInfo = await appWs.appInfo();
-  t.equal(
+  t.deepEqual(
     appInfo.status,
-    "awaiting_memproofs",
-    "app status is awaiting_memproofs"
+    { disabled: { reason: "never_started" } },
+    "app status is never_started"
```

Comment:
Was this test wrong?

Reply:
Consistently failed for me after taking a new Holochain through Nix

I know it's a recent test, should this not have changed?

Reply:
Whatever makes it happy, I'd say.

Reply:
This should have been fixed with the serialization change. But if it wasn't (and tests passed before anyway and pass now with the change), then all good.